functions/Set-DbaDbFileGrowth.ps1

```diff
@@ -72,6 +75,12 @@ function Set-DbaDbFileGrowth {
         PS C:\> Set-DbaDbFileGrowth -SqlInstance sql2017, sql2016, sql2012 -Database test -WhatIf
 
         Shows what would happen if the command were executed
+
+    .EXAMPLE
+        PS C:\> Set-DbaDbFileGrowth -SqlInstance sql2017 -Database test -GrowthType GB -Growth 1 -FileType Data
+
+        Allows you to set growth to Data or Log independently
```

Comment:
I would say "Sets a filegrowth of 1 GB for the data files only". So tell the user what the example code does and not what the used parameter can do.